.github/workflows/backport.yml

```diff
@@ -56,7 +56,7 @@ jobs:
       max-parallel: 1
       matrix:
         # 7.17 was intentionally skipped because it was added late and was bug fix only
-        target_branch: [7.13, 7.14, 7.15, 7.16, '8.0', 8.1]
+        target_branch: [7.13, 7.14, 7.15, 7.16, '8.0', 8.1, 8.2]
```

Comment:
Shouldn't we drop support for some of these?

Reply:
like we discussed, we should def consider it, but we would should do some additional review to make sure everything is aligned